addons/links/package.json

```diff
@@ -46,7 +46,6 @@
     "@storybook/core-events": "6.5.0-beta.6",
     "@storybook/csf": "0.0.2--canary.4566f4d.1",
     "@storybook/router": "6.5.0-beta.6",
-    "@types/qs": "^6.9.5",
```

Comment:
It's not always safe to move `@types/*` to devDeps. This might break consumers using strict package managers (like Yarn PnP or pnpm)

Reply:
As far I know `@types/*` packages only needed for TypeScript files, when you are `import`'ing dependencies without build-in types.

So, when I using the ` @storybook/addon-a11y` I don't want TypeScript types for React development.

Reply:
> This might break consumers using strict package managers (like Yarn PnP or pnpm)

If we talk about development of the Storybook, NPM, Yarn and Pnpm  by default will install all dependencies from a `package.json` file.

If we talk about installing a NPM package to a user project, the end user don't need `@types/*` packages that was used for development of the NPM package as nested dependencies.

Reply:
> So, when I use the @storybook/addon-a11y I don't want TypeScript types for React development.

Unless you are also a TS dev using a strict package manager AND that said `@storybook/addon-a11y` package references `'react'` somehow in its `.d.ts` (which isn't that unlikely).